packages/react-native-win32/src/Libraries/Utilities/FocusManager.win32.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+import { findNodeHandle, NativeModules } from 'react-native';
+
+/*
+** This is a helper class intended to allow usage of Polite/Aggressive Focus for win32.
+*/
+class FocusManager {
+
+    // This function takes in a ref and a bool value. If setWindowFocus = true, call aggressive focus.
+    // Else, call polite focus
+    static Focus(ref, setWindowFocus)
```

Comment:
>ref [](start = 17, length = 3)

Why is this not a typescript? It should be a TypeScipt that gets compiled into JavaScript

Reply:
@NikoAri: This is expected, this is an override of facebook code which is all in Flow.

Reply:
Interesting, but all we care here about is function signature, right? In this case, our override could be in Typescript? It's just writing directly is JavaScript will be difficult to maintain and prone to errors. Leaving it up to you though. 

---
In reply to: [539693587](https://github.com/microsoft/react-native-windows/pull/6711#discussion_r539693587) [](ancestors = 539693587)

Reply:
I'd say lets follow the pattern, all other files in the folder are flow...

Reply:
We can still add type information to this, but using Flow syntax instead of TS. The two are very similar to each other though.